java/src/jmri/jmrit/symbolicprog/SplitVariableValue.java

```diff
@@ -646,7 +644,7 @@ public void propertyChange(java.beans.PropertyChangeEvent e) {
             }
 
             if (_progState == IDLE) { // no, just a CV update
-                log.error("Variable={}; Busy goes false with state IDLE", _name);
+                log.debug("Variable={}; Busy goes false with state IDLE", _name);
```

Comment:
This really is an error in the state machine.  Did you change it because you're seeing it occurring?

Reply:
@dheap Bob actually had a concern about this, which you didn’t address.

Reply:
Sorry, missed your comment in the switching between #6921 and #6917

I saw it a couple of times and decided from the comment in the line above that it was probably benign and a result of the CV import process.

But it's a long time (years) since I worked on that part of the SplitVariableValue code.

May have been a mis-call. We can easily change with a new PR, but I need some family time out. You are welcome to revert it if you wish.